SConstruct

```diff
@@ -21,16 +21,15 @@ env.PrependENVPath("PATH", os.getenv("PATH"))
 
 # Custom options and profile flags.
 customs = ["custom.py"]
-try:
-    customs += Import("customs")
-except:
-    pass
```

Comment:
This functionality seems to be completely removed - I don't see it re-added later.

See https://github.com/godotengine/godot-cpp/pull/1196 for some background on this

Reply:
Yes, I wasn't sure how to implement that functionality. But I will wait until we get some feedback (as you have requested) before performing the change + rebase.